onyo/lib/commands.py

```diff
@@ -200,7 +201,7 @@ def onyo_cat(inventory: Inventory,
     # TODO: "Full" asset validation. Address when fsck is reworked
     assets_valid = validate_yaml(deduplicate(files))
     if not assets_valid:
-        raise OnyoInvalidRepoError("Invalid assets")
+        raise InvalidAssetError("Invalid assets")
```

Comment:
```suggestion
        e = InvalidAssetError("Invalid assets")
        e.returncode = 2
        raise e
```

Reply:
This could be done either here or in the CLI wrapper (catch the error, append the code `raise ... from`)
No need for special casing where exit anywhere other than main.

Reply:
`pyre` chokes on this, and wants the Exception to have these attributes defined.